dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/servlet/RequestUtils.java

```diff
@@ -317,12 +317,16 @@ private static String getPostBody(int len, ServletInputStream in, String encodin
             //return new Hashtable();
             // begin 157338
         }
-        // XXX we shouldn't assume that the only kind of POST body
-        // is FORM data encoded using ASCII or ISO Latin/1 ... or
-        // that the body should always be treated as FORM data.
-        //
+        
+        return postedBytes;
+    }
+    private static String getPostBody(int len, ServletInputStream in, String encoding) /* 157338 add throws */ throws IOException
+    {
```

Comment:
Add a log statement and correct method name in this block.

Reply:
I have added/fixed statements and I think the test build looks ok.